cvat/apps/documentation/user_guide.md

```diff
@@ -999,48 +998,100 @@ Press ``Esc`` to cancel editing.
-delete the previous point clicking right mouse button. Press ``N`` again for
-completing the shape. You can move points or delete them by double-clicking.
-Double-click with pressed ``Shift`` will open a polygon editor. There you can
-create new points (clicking or dragging) or delete part of a polygon closing
-the red line on another point. Press ``Esc`` to cancel editing.
+  ![](static/documentation/images/image157_mapillary_vistas.jpg)
 
-![](static/documentation/images/gif007.gif)
+- Click on the first point of the contour part.
 
-Also, you can set fixed number of points in the "poly shape size" field, then
-drawing will be stopped automatically. To enable dragging, you should right-click
-inside polygon and choose ``Enable Dragging``.
+  ![](static/documentation/images/image158_mapillary_vistas.jpg)
+
+- Then click on any point located on part of the path. The selected point will be highlighted in purple.
+
+  ![](static/documentation/images/image159_mapillary_vistas.jpg)
+
+- Сlick on the last point and the outline to this point will be built automatically.
+
+  ![](static/documentation/images/image160_mapillary_vistas.jpg)
+
+Besides, you can set a fixed number of points in the ``Number of points`` field, then
+drawing will be stopped automatically. To enable dragging you should right-click
+inside the polygon and choose ``Switch pinned property``.
 
 Below you can see results with opacity and black stroke:
 
-![](static/documentation/images/image064.jpg)
+![](static/documentation/images/image064_mapillary_vistas.jpg)
+
+If you need to annotate small objects, increase ``Image Quality`` to
+``95`` in ``Create task`` dialog for your convenience.
+
+### Make AI polygon
 
-Also, if you need to annotate small objects, increase ``Image Quality`` to
-``95`` in ``Create task`` dialog for annotator's convenience.
+Used to create a polygon semi-automatically.
+- Before starting, you have to make sure that the ``Make AI polygon`` is selected.
+
+  ![](static/documentation/images/image114.jpg)
+
+- Click ``Shape`` to enter drawing mode. Now you can start annotating the necessary area.
+  A shape must consist of 4 points minimum. You can set a fixed number of points in the ``Number of points`` field,
+  then drawing will be stopped automatically. You can zoom in/out and move while drawing.
+- Press ``N`` again to finish marking the area. At the end of Auto Segmentation,
+  a shape is created and you can work with it as a polygon.
+
+  ![](static/documentation/images/gif009_DETRAC.gif)
+
+### Edit polygon
+
+To edit a polygon you have to double-click with pressed ``Shift``, it will open the polygon editor.
+- There you can create new points or delete part of a polygon closing the line on another point.
+- After closing the polygon, you can select the part of the polygon that you want to leave.
+- You can press ``Esc`` to cancel editing.
+
+  ![](static/documentation/images/gif007_mapillary_vistas.gif)
```

Comment:
@TOsmanov , could you please check that all images are visible next time (press `view file` and inspect it)? Image file names are case sensitive. I will recommend to rename all files in lowercase.

P.S. You probably don't see the issue on Windows because file names on Windows are case insensitive. Thus you need to check the user_guide.md here.

Reply:
@nmanovic , yes it is, I corrected the file names and paths in PR #1596 , to them so that they did not contain uppercase letters.